__tests__/Auth0Client/getUser.test.ts

```diff
@@ -146,5 +148,39 @@ describe('Auth0Client', () => {
       );
       expect(user?.sub).toBe('123');
     });
+
+    it('should return from the in memory cache if no changes', async () => {
```

Comment:
Should we add a couple of extra tests to cover:

- Should update memory if changes
- Should not return anything if nothing in local storage but something in memory

Unless these are covered by already existing tests ?

Reply:
I'd added the first bullet you have but realise now that the description is incorrect. I'll split that out to a new test and then add a test for the second bullet too

Reply:
Added those two tests